### DISCUSSION TITLE
Enabling X-Frame-Options in GitHub Enterprise version

### DISCUSSION THREAD

**Discussion by gbidsilva** [community/community#24495] [Enterprise]
<p>We have a requirement to display GitHub’s OAuth2 web flow login screen / consent granting screen within an iFrame which is there in a different web application.<br>
I know by default GitHub sends the “X-Frame-Options: deny” header. Therefore, we cannot render those screen within an iFrame.</p>
<p>I want to know whether there is any way of configuring above ‘X-Frame-Option’ parameter? - so that we can use GitHub’s web interfaces within an IFrame from a different application.<br>
As I think, this might not be possible with <a href="http://github.com" rel="nofollow noopener">github.com</a> cloud version. But, Is there a way which we can allow above using GitHub Enterprise - on-premise version?</p>

**Comment by francisfuzz** [comment, marked as answer]
<p>:wave: <a class="mention" href="/gbidsilva">@gbidsilva</a>: The <code>X-Frame-Options</code> settings are not a configurable option for GitHub Enterprise. If you’re interested in sharing this use case with our product team, we highly encourage you to submit a request through <a href="https://support.github.com/contact/feedback">our official product feedback form</a> so that our product team can track it. That’s the best place to share requests like these! We also recommend following the <a href="https://github.community/t/github-roadmap-faq/124978">GitHub public roadmap</a> for the latest upcoming product updates.</p>
